### PR TITLE
chore: debugging

### PR DESCRIPTION
**Pull request by mo-cherif**
## Description

debugging docs

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -119,6 +119,18 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
 
    The app should be available at https://app.zaneops.local.
 
+
+7. **Debugging :**
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:123]: This should be its own section
```suggestion
##  🛠️ Debugging 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
...
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:123]: This should be its own section
```suggestion
##  🛠️ Debugging 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
...
+   If the app is unresponsive
+
+   0. make sure you ran make dev and it didn't exit unexpectedly
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:126]:
```suggestion
   1. make sure you ran `make dev` and it didn't exit unexpectedly 
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
 
    The app should be available at https://app.zaneops.local.
 
+
+7. **Debugging :**
+   If the app is unresponsive
+
+   0. make sure you ran make dev and it didn't exit unexpectedly 
+   1. make sure that all the containers are up, you can check it in your docker tool of choice, orbstack or docker desktop
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:127]:
```suggestion
   2. make sure that all the containers are up, you can check it in your docker tool of choice, orbstack or docker desktop 
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
...
    The app should be available at https://app.zaneops.local.
 
+
+7. **Debugging :**
+   If the app is unresponsive
+
+   0. make sure you ran make dev and it didn't exit unexpectedly 
+   1. make sure that all the containers are up, you can check it in your docker tool of choice, orbstack or docker desktop 
+   ![Alt text](https://media.discordapp.net/attachments/1146204833913782363/1234480044492198010/image.png?ex=6630e2a2&is=662f9122&hm=51504fb845b7e5f5cd36ea50908ceced025b4bbb60e9b7cf177cc53a8adff745&=&format=webp&quality=lossless&width=1880&height=1202)
+   2. make sure that the API is launched, and that no error is in thrown in the terminal where make dev is running
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:129]:
```suggestion
   3. make sure that the API is launched, and that no error is in thrown in the terminal where `make dev` is running
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
...
+   3. make sure to run nslookup
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:130]:
```suggestion
   4. make sure to run `nslookup` as stated in the previous section detailling how to setup the local domain
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
...
+   4. make sure to setup the project and install the packges with make setup
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:131]:
```suggestion
   5. make sure to setup the project and install the packages with `make setup`
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
...
+   5. If the app is still unresponsive, run make reset-db
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:132]:
```suggestion
   6. If the app is still unresponsive, run `make reset-db`
```

Also add a warning to say that this command will reset the project from scratch and delete everything

**Review by Fredkiss3 — CHANGES_REQUESTED** [review]
Changes requested

**Comment by mo-cherif** [comment]
we good now

**Review by Fredkiss3 — CHANGES_REQUESTED** [review]
importat

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -119,6 +119,19 @@ You can open a new issue with this [issue form](https://github.com/zane-ops/zane
 
    The app should be available at https://app.zaneops.local.
 
+
+##  🛠️ Debugging 
+   If the app is unresponsive
```
Comment (Fredkiss3) [inline CONTRIBUTING.md:124]:
```suggestion
   You may end up having issues where the project is not working, the app is not reachable on the browser, or the API seems to be down, this section is to help debugging this case, if the app is working fine on your end, you don't need to read this section.
```

We need a better leading paragraph here to explain what this section is.

**Review by Fredkiss3 — APPROVED** [review]
LGTM 🚀

**Pull request merged**